src/components/MultipleAvatars.js

```diff
@@ -114,12 +143,27 @@ const MultipleAvatars = (props) => {
                                 // Set overlay background color with RGBA value so that the text will not inherit opacity
                                 StyleUtils.getBackgroundColorWithOpacityStyle(themeColors.overlay, variables.overlayOpacity),
                                 styles.horizontalStackedAvatar4Overlay,
-                                StyleUtils.getAvatarBorderRadius(props.size, props.icons[3].type),
+                                StyleUtils.getHorizontalStackedOverlayAvatarStyle(oneAvatarSize, oneAvatarBorderWidth),
+                                (props.icons[3].type === CONST.ICON_TYPE_WORKSPACE ? StyleUtils.getAvatarBorderRadius(props.size, props.icons[3].type) : {}),
                             ]}
                         >
-                            <Text style={styles.avatarInnerTextSmall}>
-                                {`+${props.icons.length - 4}`}
-                            </Text>
+                            <View
+                                style={[styles.justifyContentCenter,
+                                    styles.alignItemsCenter,
+                                    {width: oneAvatarSize.width, height: oneAvatarSize.height},
```

Comment:
We shouldn't use inline styles. Ref: https://github.com/Expensify/App/blob/main/contributingGuides/STYLING.md#inline-styles